packages/documentation-site/patternfly-docs/content/get-started/about.md

```diff
@@ -10,31 +10,31 @@ PatternFly is an open source design system created to enable consistency and usa
 
 ## What's included in PatternFly?
 
-### [Design foundations](/design-foundations/about)
+### [Design foundations](/design-foundations//about-design-foundations)
```

Comment:
Is that // necessary? I'm guessing that's a typo?

Reply:
nope 😆 a typo!